src/lib/flags.js

```diff
@@ -19,5 +19,6 @@ export const initFlags = () => {
 
 const flagsList = () => {
   flag('switcher', true)
-  flag('settings.partial-desktop-sync.show-synced-folders-selection')
+  flag('settings.partial-desktop-sync.show-synced-folders-selection'),
+    flag('settings.permissions-dashboard', true)
```

Comment:
are you sure, this is working?

according to the code syntax, I'm pretty sure, it does not.

do you know how to see this flaglist? if not, let's have a look together 🙏

Reply:
> according to the code syntax, I'm pretty sure, it does not.

According to the code, I'm pretty sure it works: 
https://github.com/cozy/cozy-libs/blob/master/packages/cozy-flags/src/flag.js#L9-L17

This flaglist is just here to be called during the init phase of the app. Like that, flags are set. And flags are now displayed in the FlagSwitcher https://github.com/cozy/cozy-libs/tree/master/packages/cozy-flags#demo and now you can play with the flag by clinking on stuff instead of playing with the console.

Reply:
Apparently, not:
<img width="480" alt="Capture d’écran 2022-06-29 à 12 34 15" src="https://user-images.githubusercontent.com/8363334/176416380-801290e7-4e7a-4663-bce0-a3e2602d63ee.png">

I thought it was because of the comma, but removing it, does not change the flag seen in the flag debugger. 🤔